docs/user-guide/global-configurations/cluster-and-environments.md

```diff
@@ -33,7 +33,11 @@ Provide your kubernetes cluster’s credentials.
 
 * **Server URL**
 
-Provide the endpoint/URL of your kubernetes cluster.
+Provide the endpoint/URL of your kubernetes cluster.It is recommended to use a self-hosted URL instead of cloud hosted. Using a self-hosted URL will result in  following benefits i.e 
```

Comment:
please rephrase the sentence `Using a self-hosted URL will result in following benefits`

Reply:
> please rephrase the sentence `Using a self-hosted URL will result in following benefits`

Done